Documentation/policy/language.rst

```diff
@@ -845,6 +850,12 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
+.. note:: Layer 7 policies --and pod annotations-- result in traffic being
+   proxied through an Envoy container provided in each Cilium agent. As
+   a result, L7 traffic targeted by policies --including HTTP visibility
+   policies-- or visibility pod annotations depend on the availability of the
+   Cilium agent.
```

Comment:
It might be a bit easier to read these sentences if it is structured a bit simpler, something like:

> As a result, L7 traffic targeted by policies depend on the availability of the Cilium agent pod. This includes HTTP visibility policies and pod annotations for policy.

Arguably the last one could also be a link to the relevant section so it's clear what this statement refers to.

Reply:
Do you mean a link to pod annotations?

Reply:
yeah this one: https://docs.cilium.io/en/stable/policy/visibility/#proxy-visibility

Reply:
Is the new commit correct in this regard?